doc/developers/contributing.rst

```diff
@@ -791,7 +804,11 @@ We are glad to accept any sort of documentation:
       SelectKBest : Select features based on the k highest scores.
       SelectFpr : Select features based on a false positive rate test.
 
-  * Add one or two snippets of code in "Example" section to show how it can be used.
+  * The "Notes" section is optional. It is meant to provide information on
+    specific behavior of the class/classmethod/method.
```

Comment:
For my education; are "Notes" sections limited to "class/classmethod/method" ? e.g., could they be added to a attributes section or functions?

Reply:
For info, adding a "Notes" section using markdown inside an attribute
```
Notes
-----
```
would raise an `ERROR: Error in "rubric" directive: no content permitted.` during build, but not during numpydoc validation.
`

Reply:
Interesting, it makes sense its limited to the above list

Reply:
Who raises the error? autodoc, sphinx...?